recipe-server/client/control_new/components/common/LoadingOverlay.js

```diff
@@ -14,16 +14,30 @@ import { areAnyRequestsInProgress } from 'control_new/state/requests/selectors';
 export default class LoadingOverlay extends React.Component {
   static propTypes = {
     children: PropTypes.any,
+    condition: PropTypes.bool,
     loading: PropTypes.bool.isRequired,
+    useCondition: PropTypes.bool,
   };
 
   static defaultProps = {
     children: null,
+    condition: false,
```

Comment:
I imagined a scenario where `condition` could be `undefined`, `null`, or an actual value. I realize now, though, that I propTyped it as `bool` so it shouldn't have to worry about nulls/etc anyway. Good catch.